examples/src/main/python/ml/logistic_regression.py

```diff
@@ -28,20 +28,20 @@
 """
 A simple example demonstrating a logistic regression with elastic net regularization Pipeline.
 Run with:
-  bin/spark-submit examples/src/main/python/ml/logistic_regression.py
+  bin/spark-submit examples/src/main/python/ml/logistic_regression.py <file> <iterations>
 """
 
 if __name__ == "__main__":
 
-    if len(sys.argv) > 1:
-        print("Usage: logistic_regression", file=sys.stderr)
+    if len(sys.argv) != 3:
+        print("Usage: logistic_regression <file> <iterations>", file=sys.stderr)
```

Comment:
Now, you are requiring the caller to specify the location of the input, whereas before it would default conveniently to the sample data file. I think you need to support the existing behavior.